conans/client/cmake.py

```diff
@@ -212,6 +212,10 @@ def build(self, conan_file, args=None, build_dir=None, target=None):
         command = "cmake --build %s" % arg_list
         conan_file.run(command)
 
+    def test(self, conan_file, args=None, build_dir=None):
```

Comment:
The `test` helper should have an optional parameter to override the target. it can be custom, right?